### PR TITLE
Exception handling in cleanUp method

### PR DESCRIPTION
**Pull request by anoopvalluthadam**
This will help to write tests for Cleanup method straight forward. Otherwise I have to add Exception decorator in tests.

### CODE REVIEW

**Comment by anoopvalluthadam** [comment]
@sylvestre If this got merged, I will push this changes to other pull requests, so that coverage issue will get resolved

File: auto_nag/tests/test_query_creator.py [hunk 1]
```diff
@@ -1,35 +1,27 @@
...
         """
+        Tests for getTemplateValue,
         Expecting a VERSION number
         """
         url = "https://wiki.mozilla.org/Template:BETA_VERSION"
         beta_version = getTemplateValue(url)
         assert type(int(beta_version)) is not type(int)
 
-    def test_getReportURL(self):
+    def test_2(self):
```
Comment (sylvestre) [inline auto_nag/tests/test_query_creator.py]: Could you rename it to something more explicit?
 thanks

Comment (anoopvalluthadam) [inline auto_nag/tests/test_query_creator.py]: Sylvestre,

Nosetests will execute only alphabetically. So these methods should execute
sequentially, until and unless we give names like this, how could we
control it's execution sequentially if we give other names . I am giving
the docstring properly.

Please do let me know about your thoughts!

On Tue, Nov 3, 2015, 4:18 PM Sylvestre Ledru <notifications@github.com>
wrote:

> In auto_nag/tests/test_query_creator.py
> <https://github.com/mozilla/relman-auto-nag/pull/53#discussion_r43734624>:
>
> >          Expecting a VERSION number
> >          """
> >          url = "https://wiki.mozilla.org/Template:BETA_VERSION"
> >          beta_version = getTemplateValue(url)
> >          assert type(int(beta_version)) is not type(int)
> >
> > -    def test_getReportURL(self):
> > +    def test_2(self):
>
> Could you rename it to something more explicit?
> thanks
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/relman-auto-nag/pull/53/files#r43734624>.
>
-- 

~anoop_valluthadam

**Comment by anoopvalluthadam** [comment]
@sylvestre I did that :)

**Comment by anoopvalluthadam** [comment]
@sylvestre please let me know if there is any changes

**Comment by anoopvalluthadam** [comment]
@sylvestre any updates?

**Comment by sylvestre** [comment]
Sorry, in a work week, pretty busy

**Comment by sylvestre** [comment]
Why did you numbered the test?

**Comment by anoopvalluthadam** [comment]
Thanks man, numbers for sorting.

Nosetests used to sort methods alphabetically and execute!

On Thu, Nov 5, 2015, 3:19 PM Sylvestre Ledru <notifications@github.com>
wrote:

> Merged #53 <https://github.com/mozilla/relman-auto-nag/pull/53>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/relman-auto-nag/pull/53#event-455518028>.
>
-- 

~anoop_valluthadam

**Pull request merged**